instrumentation-genai/opentelemetry-instrumentation-vertexai/src/opentelemetry/instrumentation/vertexai/__init__.py

```diff
@@ -118,10 +123,30 @@ def _instrument(self, **kwargs: Any):
             schema_url=Schemas.V1_28_0.value,
             event_logger_provider=event_logger_provider,
         )
-
-        method_wrappers = MethodWrappers(
-            tracer, event_logger, is_content_enabled()
+        sem_conv_opt_in_mode = _OpenTelemetrySemanticConventionStability._get_opentelemetry_stability_opt_in_mode(
```

Comment:
We need to set the correct `schema_url` (semconv version) in the providers based on what version they opt-in to